lib/internal/crypto/cipher.js

```diff
@@ -109,18 +120,24 @@ function createCipherBase(cipher, credential, options, decipher, iv) {
 
 function createCipher(cipher, password, options, decipher) {
   validateString(cipher, 'cipher');
-  password = getArrayBufferView(password, 'password');
+  password = getArrayBufferOrView(password, 'password');
 
   createCipherBase.call(this, cipher, password, options, decipher);
```

Comment:
```suggestion
  FunctionPrototypeCall(createCipherBase, this, cipher, password, options, decipher);
```

Reply:
This PR intentionally does not touch most of this existing code. Would much rather do this is a separate follow on PR